lib/lotus/routing/endpoint.rb

```diff
@@ -44,6 +44,11 @@ def inspect
           "#<#{ __getobj__.class }>"
         end
       end
+
+      # @since x.x.x
+      def call(env)
```

Comment:
This isn't necessary, because `Endpoint` inherits from `SimpleDelegator` this behavior **already happens**. ;)